src/status_im/ui/screens/wallet/main/views.cljs

```diff
@@ -75,10 +77,12 @@
       (i18n/label :t/wallet-total-value)]
      [change-display change]]
     [react/view {:style (merge button.styles/buttons-container styles/buttons)}
-     [btn/button {:on-press #(rf/dispatch [:navigate-to :wallet-send-transaction])
+     [btn/button {:disabled? syncing?
+                  :on-press (when-not syncing? #(rf/dispatch [:navigate-to :wallet-send-transaction]))
```

Comment:
`disabled?` already ensures `on-press` won't be called, this check is redundant.